nixos/modules/security/pam.nix

```diff
@@ -1011,7 +1011,7 @@ in
       in
       lib.concatMapStringsSep "\n"
         (name: "r ${config.environment.etc."pam.d/${name}".source},")
-        (attrNames config.security.pam.services) +
+        (attrNames config.security.pam.services) + "\n" +
```

Comment:
maybe better:
```nix
      lib.concatMapStrings
        (name: "r ${config.environment.etc."pam.d/${name}".source},\n")
        (attrNames config.security.pam.services)
```

Reply:
Agreed! I have amended my change and also rebased off latest master.